callouts/python/README.md

```diff
@@ -4,6 +4,7 @@ Files using Copyright 2023 Google LLC & Apache License Version 2.0:
 
 * [service_callout.py](./extproc/service/callout_server.py)
```

Comment:
Should ./extproc/service/network_callout_server.py also be included in this list? If you add it here, then it's probably worth a one-line description next to each file, where you could state that callout_server.py is for HTTP-layer callouts, while network_callout_server is for network-layer callouts.